app/slackimport.go

```diff
@@ -72,6 +72,8 @@ type SlackPost struct {
 
 var isValidChannelNameCharacters = regexp.MustCompile(`^[a-zA-Z0-9\-_]+$`).MatchString
 
+const SLACK_IMPORT_MAX_FILE_SIZE = 1024 * 1024 * 50
```

Comment:
@wiersgallak  Is this a reasonable max size? I think slack imports can be much larger than this.

Reply:
@crspeller It depends on if the zip contains file attachments or not.  We support messages and file attachments.  A zip of only messages is reasonable at 50mb, but if there are attachments included the zip file could be a few gigabytes or more

Reply:
Hi @wiersgallak 

Are attachments always processed from inside `__uploads` directory inside the Zip archive? If that is the case, I can handle it as a special case and ensure the file size check is applied only for JSON files.

The core issue is parsing very large JSON files which spikes memory usage in MM server.

Reply:
Sorry for my delay on this one.  Adding @svelle as he has been assisting customers with imports and probably can help answer this much better

Reply:
I'd say due to the fact that we're migrating over to the ETL tool anyway it's okay to limit this to a value between 50-70mb for the json files.

Reply:
Hi @abihisek, would you like to modify the size that we can get it merged? Otherwise happy that we finish the patch after you did the majority of work already.

Thanks,
Daniel

Reply:
Hi @DSchalla 

Updated the max size to 70mb. Hope this works :)